README.md

```diff
@@ -146,7 +146,7 @@ c[_]
 >>> art(22,number=1) # raise artError
 Traceback (most recent call last):
         ...
-art.art.artError: artname shoud have str type
+art.art.artError: The 'artname' type must be str.
```

Comment:
Can't it be like `'artname' should be a str`?
I mean the word `type` there seems a little confusing.
In addition if we can mention the wrong `artname` in this error it will have more efficiency (here 22).